pyproject.toml

```diff
@@ -854,7 +853,12 @@ ignore = [
     "COM819",
 
     # Disabled because ruff does not understand type of __all__ generated by a function
-    "PLE0605"
+    "PLE0605",
+
+    "TC006", # disabled temporarily on ruff 0.10.0 update
+    "C420", # disabled temporarily on ruff 0.10.0 update
+    "LOG015", # disabled temporarily on ruff 0.10.0 update
+    "PLC1802", # disabled temporarily on ruff 0.10.0 update
```

Comment:
Opened two PRs to fix `C420` and `LOG015`. Those can be merged independently of the ruff update.
- #140555
- #140556

`PLC1802` is only one occurance AFAICT, so would make sense to just fix it here.